### PR TITLE
Ably #3532

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

File: components/ably/actions/publish-message/publish-message.mjs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    eventName: {
+      label: "Event Name",
+      description: "The event name that will be published",
+      type: "string",
+    },
+    messageData: {
+      label: "Message Data",
+      description: "The data of the message that will be published",
+      type: "string",
+    },
```
Comment (michelle0927) [inline components/ably/actions/publish-message/publish-message.mjs:25]: Looks like `channelName`, `eventName`, and `messageData` props are being used in both `publish-message` and `publish-push-notification`. Could we put these in the app file as propDefinitions?

**Review by michelle0927 — DISMISSED** [review]
Looks good! Just one minor nitpick.

**Review by andrewjschuang — DISMISSED** [review]
LGTM!

Just one question, is there a case in which `response.messageId` is not returned in the response?
```
    if (response.messageId) {
      $.export("$summary", `Successfully pushed notification with ID ${response.messageId}`);
    }
```

**Comment by vunguyenhung** [comment]
Hi @lcaresia, I've added a commit to fix the Publish Push Notification and verified that all test cases are passed! Ready for release! 

Test report
[Ably_3532.pdf](https://github.com/PipedreamHQ/pipedream/files/10109569/Ably_3532.pdf)

**Review by andrewjschuang — APPROVED** [review]

**Pull request merged**